languages/ruby/outline.scm

```diff
@@ -5,9 +5,21 @@
 ((identifier) @context
   (#match? @context "^(private|protected|public)$")) @item
 
-(method
-    "def" @context
-    name: (_) @name) @item
+(body_statement
+  (call
+      method: (identifier) @context
+      arguments: (argument_list
+          (method
+              "def" @context
+              name: (_) @name)
+          )) @item
+)
+
+(body_statement
+    (method
+        "def" @context
+        name: (_) @name) @item
+)
```

Comment:
Using `body_statement` here to specifically select method definitions that are not the argument to a modifier method call.